discovery-infra/tests/test_kube_api.py

```diff
@@ -264,3 +267,29 @@ def get_ignition_config_override(ca_bundle):
             }
     }
     return json.dumps(ignition_config_override)
+
+
+def collect_debug_info_from_cluster(cluster_deployment, agent_cluster_install):
```

Comment:
All the log collection logic should be defined at [download_logs.sh](https://github.com/openshift/assisted-test-infra/blob/master/scripts/download_logs.sh) or [download_logs.py](https://github.com/openshift/assisted-test-infra/blob/master/discovery-infra/download_logs.py) and not as part of a test

Reply:
sure, move to download_logs.py